apis/python/src/tiledbsoma/io/outgest.py

```diff
@@ -69,8 +69,7 @@ def to_h5ad(
     obsm_varm_width_hints: Optional[Dict[str, Dict[str, int]]] = None,
     uns_keys: Optional[Sequence[str]] = None,
 ) -> None:
-    """Converts the experiment group to `AnnData <https://anndata.readthedocs.io/>`_
-    format and writes it to the specified ``.h5ad`` file.
+    """Converts the experiment group to AnnData format and writes it to the specified ``.h5ad`` file.
```

Comment:
I really love the idea of linking to another package. Sadly, in the first-sentence-only summaries that Sphinx loves to create, it somehow can't handle these and leaves the summary _entirely blank_. Unacceptable.